javascript/javascript3/week3/gitHubPromises.js

```diff
@@ -0,0 +1,37 @@
+
+const benna100_Url='https://api.github.com/search/repositories?q=user:benna100'
+const tariqjavid_Url='https://api.github.com/search/repositories?q=user:tariqjavid'
+const mogenshoj_Url='https://api.github.com/search/repositories?q=user:mogenshoj'
+
+const  pTag=document.getElementById('pTag')
+classMatesRepositories=()=>
```

Comment:
It is not like class and also instead of calling all three users, u can have a function that gets any user and then do the functionality. then call that function three times.